test/util.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
nack,

`#!/bin/bash`

please

Reply:
For the record, I believe FreeBSD has bash in /usr/bin, if you're ever going to support the BSDs.
Sorry for hijacking :)

Reply:
@richarson I discussed it with @purpleidea in irc, I had no real argument for it, only that is was the default template in my text editor :). I believe there was no major argument against it as well. But I think having consistent conventions in the codebase is more valuable than my template's preference ;)

I'm working on a Shellcheck compliance commit, we might discus this later on there if needed or wait until someone complains about it.